testsuite/tests/kuadrant/limitador/test_basic_limit.py

```diff
@@ -34,3 +35,11 @@ def test_limit(client, limit):
         r.status_code == 200 for r in responses
     ), f"Rate Limited resource unexpectedly rejected requests {responses}"
     assert client.get("/get").status_code == 429
+
+
+def test_limit_reset(client, limit):
```

Comment:
Wouldn't this collide with the previous test when run in parallel?

Reply:
It would. Scope change to function is required. But I am inclined to remove this test.